### ISSUE TITLE
Feature: sort team/org repository listings alphabetically

### ISSUE THREAD

**Issue by ErrolSayre** [gogits/gogs#3949]
We have many repos for projects on our teams and the listings get quite long. It would be helpful to be able to sort the repositories by name rather than date added. This of course could be optional, toggled by a UI element on the listing.
* On the Team Repositories listing it would make sense to have some options in the header at the top.
* On the Org page it would be make sense to put these sort options to the left of the "New Repository" button.
* In both cases I think the minimum options would be "Sort by:"
  * Alphabetical
  * Latest activity
* It would of course be nice for each of these to toggle between ascending/descending

**Comment by MorganRamsay** [comment]
To expand this request, I would like to be able to alphabetically sort repos displayed on the user dashboard.

**Comment by alexanderadam** [comment]
Yes, I would think so. At least the first comment about default sorting in #3050 would mean that this issue here is a duplicate.
And #3841 is an exact duplicate of this one here. 😉
